assets/models/system/databricks-dolly-v2-3b/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: huggingface/databricks-dolly-v2-3b/1693842944/mlflow_model_folder
```

Comment:
model seems to be saved at different location as per run share: 

![image](https://github.com/Azure/azureml-assets/assets/61145377/cf30a076-7a19-4d74-b2ee-f8d876f28cea)